src/test/build-static.test.ts

```
@@ -0,0 +1,347 @@
+      const inlineCSS = enHTML.match(/<style[^>]*>(.*?)<\/style>/gs)?.[0] || '';
+      
+      // Classes critiques pour le Header selon l'implémentation réelle
+      expect(inlineCSS).toContain('.header-critical'); // Classe principale Header
+      expect(inlineCSS).toContain('backdrop-filter'); // Optimisations modernes
+      expect(inlineCSS).toContain('@media'); // Media queries responsive
+      expect(inlineCSS).toContain('prefers-color-scheme'); // Support dark mode
+    });
+
+    test('bundle JavaScript total < 3KB', async () => {
+      const { readdirSync, statSync } = await import('fs');
+      const { join } = await import('path');
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)